README.md

```diff
@@ -157,6 +158,13 @@ make u
 > **Note**: This is not the complete list of commands, run `make` to find out if
 > more have been added.
 
+## Environment Variables
+
+| **Environment Variable**    | **Description**                                                                                     | **Usage**                                                                                                  | **Error Handling**                                                                                                         |
+|-----------------------------|-----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| `TUS_FOCA_CONFIG_PATH`       | Specifies the path to the configuration file for the FOCA app.                                      | The FOCA app uses this environment variable to locate the configuration file. If not set, it defaults.     | If the configuration file is not found at the specified or default path, a `FileNotFoundError` will be raised.             |
```

Comment:
No need to add spacing for each field just to align the columns in edit mode. IMO it actually makes it harder to read when not rendered - and when it's rendered, it will look the same. It's also painful to maintain if you add something where a field needs more or less space in the future.

No big deal though. You can leave it if you like - up to you.